### PR TITLE
Improvements 2019-08 / AM-1234

### PR DESCRIPTION
**Pull request by tiaguinhor**

### CODE REVIEW

File: Exception/GException.php [hunk 1]
```diff
@@ -80,9 +83,8 @@ public function render($request, Exception $exception)
      */
     protected function unauthenticated($request, AuthenticationException $exception)
     {
-
         if ($request->expectsJson()) {
-            return response()->json(['error' => 'Unauthenticated.'], 401);
+            return response()->json(['error' => 'Unauthenticated.'], Response::HTTP_BAD_REQUEST);
```
Comment (inaveedahmed) [inline Exception/GException.php]: HTTP bad request = 400, Unauth should be 401

File: Exception/BadRequestException.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?php
```
Comment (inaveedahmed) [inline Exception/BadRequestException.php]: Not needed as discussed.

**Comment by inaveedahmed** [comment]
Agreed.

File: composer.json [hunk 3]
```diff
@@ -6,21 +6,35 @@
   "homepage": "https://5andhalf.com/",
   "minimum-stability": "dev",
   "prefer-stable": true,
-
   "require": {
-    "php": "^7.1.3",
+    "php": "^7.1",
     "laravel/framework": "^5.6",
```
Comment (tiaguinhor) [inline composer.json]: I'd like to update this, but we can't because of the laravel versions on the projects.

**Comment by inaveedahmed** [comment]
Merge to dev branch [here](https://github.com/inaveedahmed/gapp-laravel/commit/d64f88d4cc3fcfc3ba02ade77634810d5732cff6)

**Pull request closed without merging**